### DISCUSSION TITLE
Exclipt valence for atom

### DISCUSSION THREAD

**Discussion by ivannnnnnnnnn** [rdkit/rdkit#6809] [Q&A]
Hello. I found a case where an exception occurs when trying to create a mol from smiles. I created an [issue](https://github.com/rdkit/rdkit/issues/6795) labeled as an error, but didn't get any comments. Can anyone please advise if this is really a rdkit bug or my mistake? Thanks

> Hello. I get error with next text when try to create molecule from smiles ```FCl(F)F```:
> 
> Explicit valence for atom # 1 Cl, 3, is greater than permitted.
> ```
> mol = Chem.MolFromSmiles('FCl(F)F')
> ```
> 
> In this case 3 is valid valence for Cl

**Comment by bp-kelley** [comment]
I don't think the rdkit supports the bond types required to make chlorine tetrafluoride.  (three center-two electron bond).

While technically stable, it doesn't really come up in general rdkit usage and practice since it will set everything it touches on fire  including "cloth, wood and test engineers" and also taught me the fantastic word "hypergolic" I.e. explodes in contact with.

**Reply by ivannnnnnnnnn** [reply]
Thank you for answear
